test/test_distributions.py

```diff
@@ -70,6 +70,13 @@ def pairwise(Dist, *params):
     return Dist(*params1), Dist(*params2)
 
 
+def is_nan(tensor):
```

Comment:
`is_nan` doesn't really imply that it returns a single bool, which is True if *everything* is Nan. `is_all_nan` would be better